kDrive/AppDelegate+Launch.swift

```diff
@@ -41,6 +42,7 @@ extension AppDelegate {
             showLaunchFloatingPanel()
             requestAppStoreReview()
             askUserToRemovePicturesIfNecessary()
+            refreshCacheAndRestartUpload()
```

Comment:
When I designed `prepareRootViewController` I had a different usage in mind. For me this method should only do UI stuff

Reply:
Changed as discussed.